app/controllers/application_drafts_controller.rb

```diff
@@ -55,6 +56,16 @@ def prioritize
     redirect_to application_drafts_url
   end
 
+  def apply
+    if application_draft.ready? && application_draft.submit_application
+      flash[:notice] = 'Your application has been submitted!'
+      ApplicationFormMailer.new_application(application_draft.application).deliver
```

Comment:
Maybe use `.deliver_later` instead of `.deliver`. It uses the a background worker and is rails 4.2 compatible. Defined here: https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/config/initializers/monkey_patch_mail.rb#L6